src/ray/raylet/node_manager.cc

```diff
@@ -2258,7 +2287,7 @@ void NodeManager::ForwardTask(
 
       // Notify the task dependency manager that we are no longer responsible
       // for executing this task.
-      task_dependency_manager_.TaskCanceled(task_id);
+      task_dependency_manager_.CancelUnfinishedTask(task_id);
```

Comment:
"Unfinished" sounds like we started executing it but then didn't finish executing it, but this task has not even started to execute, right? Maybe `CancelUnexecutedTask`?